### PR TITLE
(docs) Mention required package names

### PR DESCRIPTION
**Pull request by Hofer-Julian**

### CODE REVIEW

File: TECHNICAL.md [hunk 1]
```diff
@@ -63,13 +63,15 @@ The location of the recharge cells is used to assign an recharge index by Modflo
 The file format of the .rch file is described [here](https://modflow6.readthedocs.io/en/latest/_mf6io/gwf-rch.html).
 To specify an uncoupled recharge as well, a second RCH package should be defined. 
 How to define a second stress package is explained [here](#how-to-define-a-second-stress-package-for-modflow6).
+Please note that in the model name file the [package name](https://modflow6.readthedocs.io/en/latest/_mf6io/gwf-nam.html#block-packages) has to be defined as "WELLS_MSW".
```
Comment (JoerivanEngelen) [inline TECHNICAL.md:67]: Shouldn't this be "RCH_MSW"?

File: TECHNICAL.md [hunk 2]
```diff
@@ -63,13 +63,15 @@ The location of the recharge cells is used to assign an recharge index by Modflo
...
 
 ### [filename].wel
 A dummy well file, of which the fluxes will be overrided. 
 The location of the wells is used to assign an well index by Modflow6.
 The file format of the .wel file is described [here](https://modflow6.readthedocs.io/en/latest/_mf6io/gwf-wel.html).
 To specify uncoupled extractions/injections as well, a second WEL package should be defined. 
 How to define a second stress package is explained [here](#how-to-define-a-second-stress-package-for-modflow6).
+Please note that in the model name file the [package name](https://modflow6.readthedocs.io/en/latest/_mf6io/gwf-nam.html#block-packages) has to be defined as "RCH_MSW".
```
Comment (JoerivanEngelen) [inline TECHNICAL.md:75]: And this be "WELLS_MSW"?

File: TECHNICAL.md [hunk 3]
```diff
@@ -110,7 +112,7 @@ Where `well_index` is the MODFLOW6 WEL index number, which equals the row number
 `svat` is the MetaSWAP svat number and `ly` is the Modflow layer number.
 
 ## How to define a second stress package for Modflow6
-A second stress package can be defined in the flow model's `.nam` file (GWF_1.nam).
+A second stress package `wel2` can be defined in the flow model's `.nam` file (GWF_1.nam).
```
Comment (JoerivanEngelen) [inline TECHNICAL.md:115]: Should be "e.g. `wel2`", as the name is arbitrary

File: TECHNICAL.md [hunk 4]
```diff
@@ -121,10 +123,10 @@ begin packages
...
   ic6 GWF_1/ic.ic
-  wel6 GWF_1/wel.wel wel1
+  wel6 GWF_1/wel.wel WELLS_MSW
   wel6 GWF_1/wel2.wel wel2
   sto6 GWF_1/sto.sto
   oc6 GWF_1/oc.oc
 end packages
 ```
-The optional argument values "wel1" and "wel2", specify the packagenames to be printed in the water budget .lst file.
+
```
Comment (JoerivanEngelen) [inline TECHNICAL.md:132]: I though this line of information was quite useful, so that people can look up their water budgets in the .lst file.

**Review by JoerivanEngelen — CHANGES_REQUESTED** [review]

File: TECHNICAL.md (hunk 1 above)
Comment (Hofer-Julian) [reply TECHNICAL.md:67]: Yes of course, thanks!

**Pull request merged**